### DISCUSSION TITLE
How retry the failed script?

### DISCUSSION THREAD

**Discussion by marconimedeiros2** [dataplane-app/dataplane#668] [Q&A]
I love the Dataplane and how is pratice, no-code and a great way to introduce the business employeers to data management and deployment process.

But to continue implementing it at a business enviroment we need to manage all the process mainly the inconsistences.

Example:
- Alert, email or other way to warn the users that pipeline is broken
- Options to retry, rerun or another when fail the process
- Reports for monitoring in a timescale the problems
- Etc

We have any deadline planned to it? Or I need to find it in other market solution? :(

**Comment by saul-data** [comment]
@marconimedeiros2 agreed this is very important and we are busy looking into better retry, monitoring and alerting. It’s hard to put dates on as we are also looking at Git version control of code with review and release - seems to be also a very important request by users. We will try get some of the features in over the next 6 months.
